merlin/core/compat.py

```diff
@@ -118,6 +118,11 @@ def device_mem_size(kind="total", cpu=False):
 except ImportError:
     numpy = None
 
+try:
+    import pandas
```

Comment:
where do we need to have a conditional import on pandas? Pandas is currently in the package requirements of core so should always be available.

Reply:
I think this was just for consistency